docs/source/tutorial/view_examples.rst

```diff
@@ -152,3 +152,20 @@ intended to be passed to the ``plot`` function from the ``dagre_py.core`` module
 .. image:: /code_samples/views/relation_graph.png
   :width: 400
   :alt: Relation graph visualization using dagre_py
+
+Formulations Views
+------------------
+A new method was added to the base class used by both the normal ``DataViewBuilder`` and the ``AdvancedDataViewBuilder``, that
+simplifies the configuration setup for formulation views.
+
+``add_formulation_descriptor`` will use the passed in data views client to automatically add the shares, component type
+and the name descriptors.  This requires that the dataset_ids have already been set.
+
+.. literalinclude:: /code_samples/views/data_view_builder_formulation_desc.py
+
+Ingredient Properties in AdvancedDataViewBuilder
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When defining ingredient level properties using the ``AdvancedDataViewBuilder``, you will need to define relationships
+with the ingredient property as inputs to the formulation descriptor itself.
```

Comment:
Is there only ever one relation that needs to be defined involving ingredient properties? If so, would `you will need to define relationships with the ingredient property as inputs to the formulation descriptor itself` be more accurate as `you will need to define a relation with all ingredient properties as inputs to the formulation descriptor itself`?  The motivation there being to clarify that it's a single relation to be added, not multiple relations, and that there can be more than one ingredient property.